packages/core/useCssVar/index.ts

```diff
@@ -33,7 +35,7 @@ export function useCssVar(
     variable,
     (val) => {
       if (elRef.value?.style)
-        elRef.value.style.setProperty(prop, val)
+        elRef.value.style.setProperty(propRef.value, val)
```

Comment:
```suggestion
        elRef.value.style.setProperty(unref(prop), val)
```

And we don't need the extra computed then.

Reply:
Done.